index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Displaying-and-Inputting-Data-Project</title>
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+      integrity="sha384-Bfad6CLCknfcloXFOyFnlgtENryhrpZCe29RTifKEixXQZ38WheV+i/6YWSzkz3V"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <div class="sign-in-btn">
+      <div>
+      <p>Already have an Intuit account?</p>
+      <a href="#">I forgot my User ID or Password</a>
+      </div>
+      <button type="submit" name="submit"><i class="fas fa-lock"></i> Sign In</button>
+    </div>
+    <div class="logos">
+      <img src="/images/intuit-logo-removebg-preview.png" alt="intuit-logo">
+    </div>
+    <ul class="logos-list">
+      <li><img src="/images/mint-logo-removebg-preview.png"></li> <p>mint</p>
+      <li><img src="/images/logo-qbo.png"></li> <p>quickbooks</p>
+      <li><img src="/images/png-transparent-turbotax-computer-software-tax-return-intuit-type-miscellaneous-company-trademark-removebg-preview.png"></li> <p>turbotax</p>
```

Comment:
Use an absolute path in the src attribute, it should be src="**.**/images/.......".